packages/codegen-ui-react/lib/forms/form-renderer-helper/validation.ts

```diff
@@ -29,7 +29,7 @@ export const createValidationExpression = (validationRules: FieldValidationConfi
     const elements: ObjectLiteralElementLike[] = [
       factory.createPropertyAssignment(factory.createIdentifier('type'), factory.createStringLiteral(rule.type)),
     ];
-    if ('strValues' in rule) {
+    if ('strValues' in rule && rule.strValues) {
```

Comment:
We should infer the validation rule type based on the `type` field, not on if this property exists

Reply:
Done